posthog/management/commands/test/test_sync_persons_to_clickhouse.py

```diff
@@ -106,6 +114,52 @@ def test_distinct_ids_deleted(self):
         )
         self.assertEqual(ch_person_distinct_ids, [(UUID(int=0), self.team.pk, "test-id-7", 107, True)])
 
+    def test_group_sync(self):
+        ts = datetime.utcnow()
+        Group.objects.create(
+            team_id=self.team.pk,
+            group_type_index=2,
+            group_key="group-key",
+            group_properties={"a": 1234},
+            created_at=ts,
+            version=5,
+        )
+
+        run_group_sync(self.team.pk, live_run=True, sync=True)
+
+        ch_groups = sync_execute(
+            """
+            SELECT group_type_index, group_key, group_properties, created_at FROM groups WHERE team_id = %(team_id)s
+            """,
+            {"team_id": self.team.pk},
+        )
+        self.assertEqual(len(ch_groups), 1)
+        ch_group = ch_groups[0]
+        self.assertEqual(ch_group[0], 2)
+        self.assertEqual(ch_group[1], "group-key")
+        self.assertEqual(ch_group[2], '{"a": 1234}')
+        self.assertEqual(ch_group[3].strftime("%Y-%m-%d %H:%M:%S"), ts.strftime("%Y-%m-%d %H:%M:%S"))
+
+    def test_group_sync_updates_group(self):
```

Comment:
Can we validate that running run_group_sync twice doesn't result in another update?

Reply:
good call cause it totally did 🙈